tests/stub_virtual_machine_factory.h

```diff
@@ -74,6 +74,11 @@ struct StubVirtualMachineFactory : public multipass::BaseVirtualMachineFactory
     {
         return std::make_unique<StubVMImageVault>();
     }
+
+    std::vector<multipass::NetworkInterfaceInfo> list_networks() const
+    {
+        throw multipass::NotImplementedOnThisBackendException("list-networks");
+    }
```

Comment:
Why did you need this here? Isn't it the same implementation in the base class. I must be missing something, cause compilation should have failed on the missing `override` (because of -Werror).

Reply:
Right, everything works the same way without redefining this function.